smart-contracts/src/devenv/ebrelayer.ts

```diff
@@ -114,7 +120,8 @@ export class RelayerRunner extends ShellCommand<EbrelayerResults> {
     readonly chainNet = "localnet",
     readonly ebrelayerDB = `levelDB.db`,
     readonly relayerdbPath = "./relayerdb",
-    readonly symbolTranslatorFile = "../test/integration/config/symbol_translator.json"
+    readonly symbolTranslatorFile = "../test/integration/config/symbol_translator.json",
+    readonly logFile = "/tmp/sifnode/ebrelayer.log"
```

Comment:
Lets change this to relayer.log as ebrelayer is the same codebase for the relayers and witnesses. Also while this is fine for now we will have to think about how we want to sanely handle default file names for multiple instances of the relayers/witnesses so relayer-1.log relayer-2.log from the readonly constructor.

Reply:
Let's talk; I think we need a new word